Profile Folder/chrome/locale/ru-RU/dtd/gsettings.dtd

```diff
@@ -66,8 +65,8 @@
 <!ENTITY gsettings.saveDiscardedCRXDescription	"Позволяет загрузить темы Chromium без их установки, нажав Отменить">
 <!ENTITY gsettings.toolbar						"Панель инструментов">
 <!ENTITY gsettings.needsRestart					"Необходим перезапуск">
-<!ENTITY gsettings.tabManager					"Менеджер вкладок">
-<!ENTITY gsettings.tabManagerDescription		'[!!! Éñáβℓèƨ ƭλè "ářřôω βúƭƭôñ" ωïƭλ ƭλè ℓïƨƭ ôƒ ôƥèñ ƭáβƨ ïñ ƭλè ƭáββář !!!]'>
+<!ENTITY gsettings.enableTabManager				"Включить менеджер вкладок">
```

Comment:
"Enable" should be removed from `enableTabManager` (not description), as the toggle already emplies it's for enabling.